code/__DEFINES/rust_g.dm

```diff
@@ -0,0 +1,10 @@
+// rust_g.dm - DM API for rust_g extension library
+#define RUST_G "rust_g"
```

Comment:
We already have this defined
https://github.com/ParadiseSS13/Paradise/blob/master/code/__HELPERS/_logging.dm#L2